mitodl/micromasters
# Added course price in listing

#### What are the relevant tickets?
Fixes https://github.com/mitodl/micromasters/issues/938

#### What's this PR do?
- Adds course price on dashboard.

#### Where should the reviewer start?
- see ```CourseRow.js``` and ```dashboard/api.py```

#### How should this be manually tested?
- Open MM dashboard,  add price in ecom table and see price with course on which you are not enroll

@pdpinch @giocalitri @aliceriot 
#### Screenshots (if appropriate)
Ignore upgrade button.
<img width="548" alt="screen shot 2016-09-01 at 5 58 58 pm" src="https://cloud.githubusercontent.com/assets/10431250/18168247/39849d6a-706e-11e6-9248-bc1fe8fb167d.png">

<img width="576" alt="screen shot 2016-09-02 at 11 11 51 am" src="https://cloud.githubusercontent.com/assets/10431250/18194303/19fbd036-70fe-11e6-92c1-21741d0128c8.png">
